Calibration/IsolatedParticles/interface/FindCaloHit.h

```diff
@@ -34,6 +34,8 @@ namespace spr {
   template <typename T>
   std::vector<typename T::const_iterator> findHit(edm::Handle<T>& hits, DetId thisDet, bool debug=false);
 
+  std::vector<std::vector<PCaloHit>::const_iterator> findHit(std::vector<PCaloHit>& hits, DetId thisDet, bool debug=false);
```

Comment:
Why couldn't you use the template version?
I simply don't like to add specialization for every type one uses. This is almost the same as not using templates at all.